.netlify/edge-functions-import-map.json

```diff
@@ -0,0 +1 @@
+{"imports":{"netlify:edge":"https://edge.netlify.com/v1/index.ts"}}
```

Comment:
Is it any reason why we need this file? I think we can delete the file and potentially add `.netlify` folder to `.gitignore`.

Reply:
:heavy_check_mark: